### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by jevazquezb**
In this final part of the project we:

- Applied React best practices and language style guides in code.
- Added styles to the website sections.

### CODE REVIEW

File: src/styles/book_list.css [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+svg circle:nth-child(1) {
+  stroke-dashoffset: 0;
+  stroke: #e8e8e8;
+}
+
+svg circle:nth-child(2) {
+  stroke-dashoffset: calc(440 - (207 * 64) / 100);
+
+  /* stroke: linear-gradient(to bottom, #307bbe, #379cf6);   */
```
Comment (rustam-tolipov) [inline src/styles/book_list.css]: - [ ] Kindly remove commented code from your file if it's not necessary.

File: src/styles/header.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  padding: 1.7rem 6.25vw;
+}
+
+h1 {
+  font-size: 1.875rem;
+  font-weight: bold;
+  color: #0290ff;
```
Comment (rustam-tolipov) [inline src/styles/header.css]: - [ ] I would suggest to make your Logo as clickable `cursor:pointer` and when you click if it directs you to home page it also would be good option for it.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @jevazquezb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/book_list.css (hunk 1 above)
Comment (jevazquezb) [reply src/styles/book_list.css]: Done! 😀

File: src/styles/header.css (hunk 2 above)
Comment (jevazquezb) [reply src/styles/header.css]: Done! 😀

**Review by rustam-tolipov — APPROVED** [review]
## Hello @jevazquezb👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**